src/index.html

```diff
@@ -13,6 +13,41 @@
     />
```

Comment:
The href attribute points to a '.scss' file, which is a Sass file. Browsers cannot directly interpret Sass files; they need to be compiled into CSS. Ensure that you are linking to a compiled CSS file instead.